bin/docker-image-tool.sh

```diff
@@ -44,15 +44,37 @@ function image_ref {
 function build {
   local BUILD_ARGS
   local IMG_PATH
+  local TMPFOLDER
 
   if [ ! -f "$SPARK_HOME/RELEASE" ]; then
     # Set image build arguments accordingly if this is a source repo and not a distribution archive.
+    local JARS="${SPARK_HOME}/assembly/target/scala-${SPARK_SCALA_VERSION}/jars"
+    TMPFOLDER=`mktemp -q -d examples.XXXXXX`
+    if [ $? -ne 0 ]; then
+      ehco "Cannot create temp folder, exiting..."
+      exit 1
+    fi
+
+    mkdir -p "${TMPFOLDER}/jars"
+    cp "${SPARK_HOME}"/examples/target/scala*/jars/* "${TMPFOLDER}/jars"
```

Comment:
Do we need the temp folder? Can we just pass the path `"${SPARK_HOME}"/examples/target/scala*/jars/*` to the dockerfile instead to pick up?

Reply:
We cannot as long as we do not want to delete from the target directory like done in [this line](https://github.com/jerryshao/apache-spark/blob/e9ea7e5dc0cd2c3456112ad46c754571ac6e555b/bin/docker-image-tool.sh#L63) where the duplicated jars are removed.
`